ui/src/DataService.js

```diff
@@ -43,6 +43,11 @@ const request = async (path, method, body) => {
  */
 const fetchItems = async url => {
   const result = await request(url, 'GET');
+  // if a 204, return an empty array
+  if (result === true) {
```

Comment:
should this not be if result is false?

Reply:
Look at the `request` function - it is returning true if the response is 204.

Reply:
Ok, understand now :+1: